tests/hpc/spack_master.pm

```diff
@@ -41,7 +43,9 @@ sub run ($self) {
 
     barrier_wait('MPI_SETUP_READY');
     if (check_var('HPC_LIB', 'boost')) {
-        assert_script_run 'spack load boost';
+        assert_script_run 'source /usr/share/spack/setup-env.sh';
```

Comment:
I suppose we will always use `/bin/sh` as symlink to `bash`, but `. /usr/share/spack/setup-env.sh` would work also on `dash`.

Reply:
AFAIK the first is true and we really dont use dash. lets leave it as it is.